src/server/db/story-dbConfiguration.js

```diff
@@ -115,8 +126,18 @@ sjrk.storyTelling.server.storiesDb.storiesByIdFunction = function (doc) {
     emit(doc._id, browseDoc);
 };
 
+/**
+* This function is used to validate new documents once it's migrated to CouchDB
+* For more info on this process, please see the CouchDB guide:
+* https://guide.couchdb.org/editions/1/en/validation.html
+*
+* @param {Object} newDoc - the incoming doc
+* @param {Object} oldDoc - pre-existing doc on the disk, if any exists
+* @param {Object} userCtx - the user and their roles
+* @param {Object} secObj - the Security Object
+*/
 sjrk.storyTelling.server.storiesDb.validateFunction = function (newDoc, oldDoc, userCtx, secObj) {
```

Comment:
The last 2 arguments are defined but never used in the function. Shall they be removed?

Reply:
I would be helpful to improve the linting to catch this type of issue.

Reply:
I believe I left them in thinking the database would complain if they weren't present, but I've just tested the function with only the first arg present (only `newDoc` is referred to in the function body) and everything seems to be working properly. The linter did in fact catch this back when I originally added the function, but since I believed those args to be necessary, I added those arg names to the global eslint definition (line 14). I've now removed them.

That should make things a little easier to document :)